ietf/iesg/tests.py

```diff
@@ -182,6 +185,1578 @@ def test_ietf_activity(self):
+
+        self.assertEqual(
+            ad_totals,
+            {
+                "ad_group_count": 17,
+                "doc_group_count": 1,
+                "doc_count": 1,
+                "page_count": 9,
+            },
+        )
+        self.assertEqual(
+            noad_totals,
+            {
+                "ad_group_count": 17,
+                "doc_group_count": 0,
+                "doc_count": 0,
+                "page_count": 0,
+            },
+        )
+        self.assertEqual(
+            totals,
+            {
+                "group_count": 17,
+                "doc_count": 1,
+                "page_count": 9,
+                "groups_with_docs_count": 1,
+            },
+        )
+
+        count_sax_dicts = 0
+        for d in wg_summary:
+            if d["wg"] == "saxophone":
+                count_sax_dicts += 1
+                self.assertEqual(d["doc_count"], 1)
+                self.assertEqual(d["page_count"], 9)
+            else:
+                self.assertEqual(d["doc_count"], 0)
+                self.assertEqual(d["page_count"], 0)
+        self.assertEqual(count_sax_dicts, 1)
+
+        # Now give Lima a document in a group that's not in their area:
+        WgDraftFactory(
+            group=Group.objects.get(acronym="gong"),
+            pages=len("gong"),
+            ad=Person.objects.get(name="Lima"),
+        )
+        (
+            area_summary,
+            area_totals,
+            ad_summary,
+            noad_summary,
+            ad_totals,
+            noad_totals,
+            totals,
+            wg_summary,
+        ) = get_wg_dashboard_info()
+        seen_dicts = Counter([d["area"] for d in area_summary])
+        for d in areas:
+            self.assertEqual(seen_dicts[area], 1 if area in ["violet", "green"] else 0)
+        for d in area_summary:
+            if d["area"] in ["violet", "green"]:
+                self.assertEqual(d["doc_count"], 1)
+                self.assertEqual(d["page_count"], 9 if d["area"] == "violet" else 4)
+                self.assertEqual(d["group_percent"], 50)
+                self.assertEqual(d["doc_percent"], 50)
+                self.assertEqual(
+                    d["page_percent"],
+                    100 * 9 / 13 if d["area"] == "violet" else 100 * 4 / 13,
+                )
+            else:
+                self.assertEqual(d["doc_count"], 0)
+                self.assertEqual(d["page_count"], 0)
+                self.assertEqual(d["group_percent"], 0)
+                self.assertEqual(d["doc_percent"], 0)
+                self.assertEqual(d["page_percent"], 0)
+
+        self.assertEqual(
+            area_totals, {"group_count": 2, "doc_count": 2, "page_count": 13}
+        )
+
+        for d in ad_summary:
+            if d["ad"] == "Lima":
+                self.assertEqual(d["doc_group_count"], 1)
+                self.assertEqual(d["doc_count"], 1)
+                self.assertEqual(d["page_count"], 9 if d["area"] == "violet" else 4)
```

Comment:
You're not counting `dicts` in this block like you did in others - is something constraining the test to be sure that both branches of this assertion are visited?